src/components/repository-profile.component.js

```diff
@@ -123,6 +123,8 @@ export const RepositoryProfile = ({ repository, starred, navigation }: Props) =>
       </Text>
 
       <Text
+        /* @see https://github.com/gitpoint/git-point/issues/114 */
```

Comment:
I initially wanted to use if we truncate the string, but than found the `numberOfLines` option. I think yeah, we can remove it. Done in 4b55d88